Resources/skeleton/crud/actions/delete.php.twig

```diff
@@ -5,51 +6,44 @@
      *
 {% block phpdoc_method_annotations %}
 {% if 'annotation' == format %}
-     * @Route("/{id}", name="{{ route_name_prefix }}_delete")
+     * @Route("/{id}", requirements={"id" = "\d+"}, name="{{ route_name_prefix }}_delete")
```

Comment:
I think we should remove all of these. They're not really needed (the query will fail if you pass something crazy). And, if you use a "guid" as your id (Doctrine supports this), then it won't be an integer :)